### PR TITLE
[Feature] applied table overview with new design

### PR DESCRIPTION
**Pull request by neighborhood999**
## Screenshots

**Single**

https://user-images.githubusercontent.com/10325111/187808136-c907f6a7-f72d-46f7-b54c-8e7889b12703.mp4

**Comparison**

https://user-images.githubusercontent.com/10325111/187808163-d1001df3-0628-425a-b02a-3f9ff3439d88.mp4

## Description

- Applied table overview with new design
- Changed naming from test to assertion made it accurately
- Removed some e2e tests cases due to UI changed
- Removed event tracking due to UI changed

Signed-off-by: Jie Peng <im@jiepeng.me>

**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signedSigned-off-by: Jie Peng <im@jiepeng.me>Signed-off-by: Jie Peng <im@jiepeng.me>

### CODE REVIEW

File: static_report/src/components/SingleReport/SRAssertionDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+          <Tbody>
+            {piperiderTableAssertions.map((tableAssertion) => {
+              zReport(assertionTestSchema.safeParse(tableAssertion));
+              const isFailed = tableAssertion.status === 'failed';
+
+              return (
+                <Tr key={tableAssertion.name}>
+                  <Td>{tableAssertion.name}</Td>
+                  <Td>
+                    {isFailed ? (
```
Comment (jonycfu) [inline static_report/src/components/SingleReport/SRAssertionDetails.tsx:64]: can we use `static_report/src/components/shared/TestStatus.tsx` ? seems to do same thing for these render blocks

File: static_report/cypress/e2e/comparison-report.cy.ts [hunk 2]
```diff
@@ -49,15 +49,6 @@ describe('Comparison Report', () => {
     cy.url().should('equal', 'http://localhost:3001/#/');
   });
```
Comment (jonycfu) [inline static_report/cypress/e2e/comparison-report.cy.ts:51]: Add test case?:
Would it be valuable to test the collapse content function here? 
I'm thinking about scenario where something is off with the data and user cannot expand/collapse.

Comment (neighborhood999) [reply static_report/cypress/e2e/comparison-report.cy.ts:51]: When the assertion is empty, it cannot expand/collapse, but our e2e test data didn't cover this.

I added a prop to check whether the assertion exists or not and decide whether the content is collapseable.

Comment (jonycfu) [reply static_report/cypress/e2e/comparison-report.cy.ts:51]: I feel like that prop isn't needed, as we should avoid adding test "props" in our components. 
I was more thinking that we can check the height value, which is what chakra uses 
image
image

Comment (neighborhood999) [reply static_report/cypress/e2e/comparison-report.cy.ts:51]: hmm, you're right. I think we need to reveal parts of information to users, so we can utilize `startingHeight` to adjust block height, especially assertions is empty we need to display no assertions content to tell users.

**Review by jonycfu — COMMENTED** [review]
`static_report/src/components/SingleReport/SRAssertionDetails.tsx`
Still can reuse AssertionStatus.

**Review by jonycfu — APPROVED** [review]

**Pull request merged**